extensions/widgets/browser/browser.lcb

```diff
@@ -24,14 +24,14 @@ Licensed under the terms of the Creative Commons Attribution 3.0 Unported Licens
 /*
 This widget displays web content within a native web browser view.
 
-Name: url
+Name: URL
 Type: property
 
 Syntax:
-set the url of <widget> to <pUrl>
-get the url of <widget>
+set the URL of <widget> to <pUrl>
```

Comment:
Okay - so 'url' is somewhat inconsistent in LCS at the moment... I vote for normalizing around the standard convention which is camel-case - so 'url', not 'Url' or 'URL'. (Things are case-insensitive anyway, however we need to make sure we keep to a convention).